engine/engine-script-library

```diff
@@ -305,14 +305,20 @@ function validate_core_env() {
     if [ -z "$cs_label" ]; then
         exit_error "The client/server label (--cs-label) was not defined"
     fi
-    # profiler-remotehost-1-sysstat-1
-    regex1='^profiler-\w+-\d+-\w+-\d+$'
-    # client-1
-    regex2='^\w+-\d+$'
-    if echo $cs_label | grep -q -P $regex1; then
+    # (rfolco): regex 101
+    #     ^profiler- ensures the string starts with profiler-.
+    #     ([a-z]+) captures one or more lowercase letters for the second part.
```

Comment:
We should probably do `a-zA-Z` everywhere you have `a-z` now so that capital letters don't break it.

Reply:
ack

Reply:
Can you update the comments also?